x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -234,6 +306,10 @@ export const ComplianceDashboard = () => {
   });
 
   useEffect(() => {
+    if (hasUserSelectedTab) {
```

Comment:
another addition is not to select the default tab after a user's interaction with tabs. With the refresh logic it is confusing when tabs are jumping around. Eg. when the user has CSPM data, but onboarding to KSPM, the tab keeps defaulting to CSPM with every status update made in the background